src/common/components/focus-component.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { VisualizationConfiguration } from 'common/configs/visualization-configuration';
+import { VisualizationStoreData } from 'common/types/store-data/visualization-store-data';
+import { WindowUtils } from 'common/window-utils';
+import { TabStopRequirementActionMessageCreator } from 'DetailsView/actions/tab-stop-requirement-action-message-creator';
+import * as React from 'react';
+
+export interface FocusComponentProps {
```

Comment:
Some of these props feel more like `deps`; here are some examples of [windowUtils](https://github.com/microsoft/accessibility-insights-web/search?l=TSX&q=windowutils) and [tabStopRequirementActionMessageCreator](https://github.com/microsoft/accessibility-insights-web/search?l=TSX&q=tabStopRequirementActionMessageCreator) usage. Functionally `deps` still are passed through the `props` chain, but we often separate them in our codebase. Usually `deps` includes methods/classes whereas other `props` include more statelike data. This allows us to instantiate `deps` once at the root level:

https://github.com/microsoft/accessibility-insights-web/blob/18832555826b0666423c956529749ba6b05cd40b/src/DetailsView/details-view-initializer.ts#L488

and only reference them in components that use them directly (example in `Toast.tsx`):

https://github.com/microsoft/accessibility-insights-web/blob/18832555826b0666423c956529749ba6b05cd40b/src/common/components/toast.tsx#L9

Then components in between who are just passing those `deps` along can reference child component `dep` types to construct their own (example in `copy-failure-issues-details-button`):

https://github.com/microsoft/accessibility-insights-web/blob/0cf0164cc07552761ca5010f2071e3cc891a1527/src/common/components/copy-issue-details-button.tsx#L13

Following the same pattern here would allow you to remove the explicit reference to `windowUtils` in `AdhocTabStopsTestView`.